text-to-speech/src/main/java/com/ibm/watson/developer_cloud/text_to_speech/v1/TextToSpeech.java

```diff
@@ -275,6 +274,26 @@ public TextToSpeech(String username, String password) {
     final boolean isNew = model.getId() == null;
     final String path = isNew ? PATH_CUSTOMIZATIONS : String.format(PATH_CUSTOMIZATION, model.getId());
 
+    // For update, a specified language cannot differ from existing model.
+    if (!isNew && model.getLanguage() != null) {
+      final Request request = RequestBuilder.get(path).build();
+      createServiceCall(request, new ResponseConverter<CustomVoiceModel>() {
+
+        // The existing model can be null for unit test.
+        @Override
+        public CustomVoiceModel convert(Response response) {
+          CustomVoiceModel existing = ResponseUtils.getObject(response, CustomVoiceModel.class);
+
+          if (existing != null) {
+            Validator.isTrue(model.getLanguage().equals(existing.getLanguage()),
+                             "cannot change language of existing model");
+          }
+
+          return null;
+        }
+      }).execute();
+    }
```

Comment:
What's the error that the user will get if they try to change the language? I feel like you are catching the error in the application but it should be the service the one returning this error.